streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -1569,7 +1569,7 @@ private void assertNextOutputRecord(final TestRecord<String, String> record,
     }
 
     private StreamPartitioner<Object, Object> constantPartitioner(final Integer partition) {
```

Comment:
Can we improve the generic types?

Reply:
@caioguedes Any thoughts?

Reply:
Sure, most of the test cases are using `StringDeserializer`, it makes sense move to `StreamPartitioner<String, String>`